services/scripts/testy-snapshot.sh

```diff
@@ -82,20 +98,27 @@ main() {
 
     # Validate database. Update the snapshot status on success/failure.
     aws ec2 create-tags --resources "$_snapshot_id" "$_volume_id" \
-                        --tags Key=Validation,Value=none
+                        --tags Key=Validation,Value=none Key=InstanceID,Value="$_instance_id"
 
     echo "Running validation script '$_validation_script' on volume '$_volume_id'."
-    var=$(date +%s%3N)
+    ts=$(date +%s%3N)
     if validate_database "$_validation_script" "$_mount_point" "$_snapshot_id" "$_volume_id"; then
         echo "Successfully validated database backup snapshot '$_snapshot_id'."
         aws logs put-log-events --log-group-name testy-logs \
                                 --log-stream-name testy-logs --log-events \
-            timestamp=$var,message="Backup snapshot validation succeeded. $_snapshot_id. $_instance_id"
+            timestamp=$ts,message="Backup snapshot validation succeeded. $_snapshot_id. $_instance_id"
+        # We can delete the snapshot now it has been validated.
+        echo "Deleting snapshot '$_snapshot_id' ..."
+        if aws ec2 delete-snapshot --snapshot-id "$_snapshot_id"; then
+            echo "Deleted snapshot '$_snapshot_id'."
+        else
+            echo "Error: Failed to delete snapshot '$_snapshot_id'."
+        fi
     else
         echo "Validation failed for database backup snapshot '$_snapshot_id'."
         aws logs put-log-events --log-group-name testy-logs \
                                 --log-stream-name testy-logs --log-events \
-            timestamp=$var,message="Backup snapshot validation failed. $_snapshot_id. $_instance_id"
+            timestamp=$ts,message="Backup snapshot validation failed. $_snapshot_id. $_instance_id"
```

Comment:
maybe we could say "instance: $_instance_id"" since it might not be obvious the random printed number is instance id. I believe snapshot id has "snapshot-123" so we should be fine there.

Reply:
This applies to all the above backup validation logs

Reply:
I have updated the messages in 9c739d5c22b1ccfe6746c84c2bcc0176548a71e9